lib/models/test-utils/testModelsHelper.js

```diff
@@ -0,0 +1,10 @@
+import User from '../user';
+import { userFixture } from './fixtures/user.fixtures';
+import Organisation from '../organisation';
+import { orgFixture } from './fixtures/organisation.fixtures';
+
+export const createUser = async () =>
```

Comment:
Here you should pass `user` as an argument to this method. It's bad idea to simply use predefined fixture by default

Reply:
Done!